src/java/org/apache/cassandra/db/guardrails/GuardrailsConfig.java

```diff
@@ -46,13 +46,6 @@
  */
 public interface GuardrailsConfig
 {
-    /**
-     * Whether guardrails are enabled or not.
-     *
-     * @return {@code true} if guardrails are enabled, {@code false} otherwise
-     */
-    boolean getEnabled();
```

Comment:
There is a comment mentioning the global enable/disable flag it the class' JavaDoc, [here](https://github.com/apache/cassandra/blob/26861b173f172b663a86e03a835ac03038be13d1/src/java/org/apache/cassandra/db/guardrails/GuardrailsConfig.java#L33-L34), that should be updated. It could be updated to something like `This contains individual settings to control each guardrail.`. That same comment also exists on [`GuardrailsOptions`](https://github.com/apache/cassandra/blob/26861b173f172b663a86e03a835ac03038be13d1/src/java/org/apache/cassandra/config/GuardrailsOptions.java#L45-L46).